source/material-terms.ttl

```diff
@@ -0,0 +1,68 @@
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix : <https://id.kb.se/vocab/> .
+@base <https://id.kb.se/term/rda/material/> .
```

Comment:
Or just `</material>`? And coordinate all the things (getty, maps/gmgpc...). 😉

Reply:
If so I think we should either (1) not bring along RDA-definitions in the construct or (2) model the form to clearly indicate the source of each definition.

Reply:
We use `skos:exactMatch` here, and pick the english labels and definitions by choice. And presumably we will extend our labels with swedish definitions as well; we could also provide alterered english definitions and only fallback to the ones from the RDA sources).

Of course, we could augment this further, (using `derivedFrom` on the resulting records), but we haven't done so for the other interlinked datasets generated by `definitions` (such as languages).

Reply:
I started to think of using `exactMatch` and `closeMatch` as an indicator for proximity but that felt a little bit like semantic abuse. `derivedFrom` could of course be a possibility but becomes blurred if we have several sources. How is that qualifier Ontology coming along? :)

Reply:
We use `exactMatch` for such qualifications already in definitions, and use that to "imply" that this is (may be) derived from the descriptions of the matches (as they are evidently "coordinated to" by us). Good enough? (We can always extend with clear record ("data document") provenance to state record-to-record relations; and in the future even star-annotations per piece if we want (as in need) to go there.)